### ISSUE TITLE
Allow exporting to one JSON file per language

### ISSUE THREAD

**Issue by tcitworld** [jshmrtn/vue3-gettext#17]
Using `gettext-compile` it was possible to export each language in their own JSON file, so that it can be dynamically loaded, something like:

```makefile
@for lang in $(LOCALES); do \
        gettext-compile --output $(OUTPUT_DIR)/translations/$$lang.json $(OUTPUT_DIR)/locale/$$lang/app.po; \
done;
```

In `next` `gettext_compile.ts` doesn't support that. This could be handled with a `split` option under `output` in `gettext.config.js`.

**Comment by lzurbriggen** [comment]
Hi @tcitworld, I just released [2.2.0-alpha.1](https://www.npmjs.com/package/vue3-gettext/v/2.2.0-alpha.1) with this feature.

Enable it in the config using `config.output.splitJson` and make sure your `config.output.jsonPath` (if set) ends with a directory, not a json file.

The locale json files container their own locale keys, so postloading could work like this:
```ts
import italian from "./it_IT.json";

const language = useGettext();
language.translations = {
    ...language.translations,
    ...italian,
};
```

Please test this prerelease and let me know if it works for you.

**Comment by lzurbriggen** [comment]
Released with [2.2.0](https://github.com/jshmrtn/vue3-gettext/releases/tag/v2.2.0)
